definitions/infra-container/definition.yml

```diff
@@ -3,25 +3,38 @@ type: CONTAINER
 
 synthesis:
   rules:
+  # Dockerstats receiver from otel collector.
   - identifier: container.id
     name: container.name
     encodeIdentifierInGUID: true
-    # This is a temporary condition to restrict synthesis for a particular set of telemetry data.
     conditions:
-      - attribute: nr.entity_type
-        value: container
+      - attribute: eventType
+        value: MetricRaw
+      # All metrics from the receiver starts with the 'container.' prefix.
+      - attribute: metricName
+        prefix: container.
+      - attribute: instrumentation.provider
+        value: opentelemetry
+      # This filters only metrics comming from docker receiver, given that metrics
+      # could deffer between different runtime receivers. 
+      - attribute: otel.library.name
+        value: otelcol/dockerstatsreceiver
     tags:
+      # Default resource attributes
       container.runtime:
-      container.image.name:
-      container.image.tag:
```

Comment:
any reason you removed this one?

Reply:
yes, `container.image.tag` is not an [attribute from the dockerstats receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/metadata.yaml#L12). I removed to avoid confusion.

Reply:
If I'm not wrong it's defined [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/62c39d6ba132ed5377620a53eebe6a265b5f5f3d/receiver/dockerstatsreceiver/metadata.yaml#L21) 🤔